bin/functions

```diff
@@ -166,18 +85,29 @@ function install {
 
     ${SHA256SUM} -c <<<"${checksum} ${package_filename}"
 
-    tar xf "${package_filename}"
+    case "${package_filename}" in
+        *zip) unzip "${package_filename}"
+              ;;
+        *tar.gz) tar xf "${package_filename}"
+                 ;;
+        *) echo "Cannot extract ${package_filename}"
+           exit 1
+           ;;
+    esac
```

Comment:
We could probably also make `unzip` an optional dependency which would show more installable candidates.

Some OpenJDK distributions are only available as ZIP archives for macOS, such as Liberica.